### ISSUE TITLE
Question 31

### ISSUE THREAD

**Issue by komalvhanmane** [Google-Developer-Student-Club-CCOEW/Competitive-Coding#53]
Let us call an integer interesting if the product of its digits is [divisible](https://en.wikipedia.org/wiki/Divisor#Definition) by the sum of its digits. You are given two integers A and B. Find the number of interesting integers between A and B (both inclusive).

Input
The first line of the input gives the number of test cases, T. T lines follow.

Each line represents a test case and contains two integers: A and B.

Output
For each test case, output one line containing Case # x: y, where x is the test case number (starting from 1) and y is the number of interesting integers between A and B (inclusive).

Input :
4
1 9
91 99
451 460
501 1000

Output :
Case #1: 9
Case #2: 0
Case #3: 5
Case #4: 176

**Comment by sayli2003** [comment]
Sayli Borole
CCOEW
Comp Sy
I would like to work on this issue

**Comment by komalvhanmane** [comment]
@niteshjeganathan Please mention your college , name , year , branch.

**Comment by komalvhanmane** [comment]
@sayli2003 Please create your pr for previously assigned issue :) . Once it is merged then only I will be able to assign you another issue.

**Comment by niteshjeganathan** [comment]
Vellore Institute of Technology, Vellore. Nitesh Jeganathan. 2nd year. CSE CORE

**Comment by komalvhanmane** [comment]
@niteshjeganathan I have assigned you this issue . Work on it :)

**Comment by niteshjeganathan** [comment]
Hey, I addressed this Question at PR #65 . Please do accept it @komalvhanmane
